css/basic.css

```diff
@@ -2,6 +2,7 @@
 
 * {box-sizing: border-box}
 :root{
+--serif:1em/1.6 system-ui, -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Oxygen, Ubuntu, Cantarell, Droid Sans, Helvetica Neue, Fira Sans, sans-serif;
```

Comment:
It should be  --sans-serif or just --sans

Reply:
done